wondrous-app/components/Common/NewInviteLinkModal/styles.tsx

```diff
@@ -0,0 +1,374 @@
+    font-size: 20px;
+    color: #fff;
+  }
+`;
+
+export const TextSubheading = styled(Typography)`
+  && {
+    font-weight: bold;
+    font-size: 20px;
+    color: #ccbbff;
+  }
+`;
+
+export const CloseButton = styled(ModalCloseButton)`
+  width: 34px;
+  height: 34px;
+`;
+
+export const StyledDivider = styled(Divider)`
+  && {
+    background: #363636;
+    height: 1px;
+    border-radius: 6px;
+    width: 630px;
+    margin: 30px auto 0;
+  }
+`;
+
+export const InviteThruLinkLabel = styled(Typography)`
+  && {
+    color: #ccbbff;
+    font-size: 14px;
+    font-weight: 500;
+    margin-top: 32px;
+  }
+`;
+
+export const InviteThruLinkInputWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  gap: 16px;
+  margin-top: 12px;
+`;
+
+export const InviteThruLinkTextField = styled(TextField)`
+  background: #0f0f0f;
+  border-radius: 6px;
+  height: 40px;
+  width: 351px;
+  padding: 0;
+
+  .MuiInputBase-input {
+    color: #c4c4c4;
+  }
+
+  .MuiOutlinedInput-input {
+    padding: 11px 15px;
+  }
+
+  &::after {
+    content: '';
+    position: absolute;
+    inset: 0;
+    border-radius: 6px;
+    background: linear-gradient(270deg, #0f0f0f 2.96%, rgba(15, 15, 15, 0) 62.5%);
+  }
+`;
+
+export const InviteThruLinkFormControlSelect = styled(FormControl)``;
+
+export const StyledSelect = styled(Select)`
+  && {
+    background: #0f0f0f;
+    width: 141px;
+    max-width: 100%;
+    height: 40px;
+    color: #fff;
+    border-radius: ${(props) => (props.rounded ? '6px' : '0')};
+    font-size: 14px;
+    text-transform: ${(props) => (props.capitalize ? 'capitalize' : 'inherit')};
+  }
+
+  & .MuiSelect-root {
+    padding-left: 12px;
+  }
+
+  & .MuiInputBase-root {
+    border-radius: 0 6px 6px 0;
+  }
+
+  & .MuiInput-underline {
+    :hover:not(.Mui-disabled)::before {
+      border: none;
+      ::before {
+        border: none;
+      }
+      ::before {
+        border: none;
+      }
+    }
+  }
+
+  & .MuiSvgIcon-root {
+    color: ${(props) => props.svgColor};
+  }
+`;
+
+export const InviteThruLinkSelect = styled(({ className, ...props }) => (
+  <StyledSelect
+    {...props}
+    capitalize={true}
+    svgColor="#CCBBFF"
+    rounded={true}
+    MenuProps={{ classes: { paper: className } }}
+  />
+))`
+  &.MuiPaper-root {
+    background: linear-gradient(180deg, #1e1e1e 0%, #141414 109.19%);
+    width: 141px;
+    color: #fff;
+  }
+  &.MuiPaper-root > .MuiList-padding {
+    padding: 0;
+  }
+`;
+
+export const InviteThruLinkMenuItem = styled(MenuItem)`
+  && {
+    background: #121212;
+    color: #c4c4c4;
+    margin: 6px;
+    border-radius: 6px;
+    font-size: 13px;
+    border: 1px solid transparent;
+    height: 34px;
+  }
+
+  &&:hover {
+    background: rgba(0, 0, 0, 1);
+    border: 1px solid #7427ff;
+  }
+
+  .MuiListItem-root.Mui-selected,
+  .MuiListItem-root.Mui-selected:hover {
+    background: none;
+  }
+`;
+
+export const InviteThruLinkButton = styled(Button)`
+  && {
```

Comment:
Can you use this common button `components/Button/index.tsx` or create a new one (here `components/Button/index.tsx`)  to prevent duplicated code?